<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CurveWidget</class>
 <widget class="QWidget" name="CurveWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>776</width>
    <height>346</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>512</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16666</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>9</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="minimumSize">
          <size>
           <width>78</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>78</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Datapoints</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <layout class="QHBoxLayout" name="Header">
           <item>
            <widget class="QLabel" name="label_01">
             <property name="text">
              <string>Distance</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_02">
             <property name="text">
              <string>Volume</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="datapoints_layout">
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Orientation::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <property name="spacing">
            <number>16</number>
           </property>
           <property name="leftMargin">
            <number>8</number>
           </property>
           <property name="rightMargin">
            <number>8</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QPushButton" name="add_data_point_button">
             <property name="styleSheet">
              <string notr="true">
    /* QPushButton default and hover styles */
    QPushButton {

        font: 580 9pt &quot;Segoe UI&quot;;
	

        border: 2px solid black;
        border-radius: 2px;
        border-color: rgba(80, 80, 80, 255);
        height:22px;
        padding-top: 2px;
        padding-bottom:2px;
        padding-left: 4px;
        padding-right: 4px;
        color: #E3E3E3;
        background-color: #1C1C1C;
    }
    QPushButton:hover {
        background-color: #414956;
        color: white;
    }
    QPushButton:pressed {
        background-color: red;
        background-color: #1C1C1C;
        margin: 1 px;
        margin-left: 2px;
        margin-right: 2px;

    }</string>
             </property>
             <property name="text">
              <string>Add datapoint</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="advanced_mode_checkBox">
             <property name="styleSheet">
              <string notr="true">
    /* QPushButton default and hover styles */
    QCheckBox {

        font: 580 10pt &quot;Segoe UI&quot;;

        border: 2px solid black;
        border-radius: 2px;
        border-color: rgba(80, 80, 80, 255);
        height:22px;
        padding-top: 2px;
        padding-bottom:2px;
        padding-left: 4px;
        padding-right: 4px;
        color: #E3E3E3;
        background-color: #1C1C1C;
    }
    QCheckBox:hover {
        background-color: #414956;
        color: white;
    }
    QCheckBox:pressed {

    }</string>
             </property>
             <property name="text">
              <string>Advanced mode</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="minimumSize">
          <size>
           <width>78</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>78</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Preview</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QLabel" name="preview_label_01">
               <property name="text">
                <string>Distance</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="preview_label_02">
               <property name="text">
                <string>Volume</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_8">
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QGraphicsView" name="graphicsView_01"/>
             </item>
             <item>
              <widget class="QGraphicsView" name="graphicsView_02"/>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../resources.qrc"/>
 </resources>
 <connections/>
</ui>
